### DISCUSSION TITLE
Might want to consider removing "EPG for IPTV" aka epg.best from Resources section

### DISCUSSION THREAD

**Discussion by 36mafia** [iptv-org/community#326] [Ideas]
This is a reply from one of their employees.

<img width="972" alt="Screenshot 2023-03-15 at 1 50 23 AM" src="https://user-images.githubusercontent.com/37257836/225256549-acf17d53-58de-4dfa-8af0-f4062646f8e4.png">

**Comment by 36mafia** [comment]
To add context, the way they have their pricing structure displays the monthly price in a much larger font. If you look at the first one, it is billed monthly, which is what misled me when I clicked on the bottom one for $3.22 a month for 500 channels.

Secondly, I was on day 1 of my 5 day trial and had watched the tutorials but still wasn't able to get the site to generate an EPG for my sources. I don't recall seeing the total price before I paid, since I don't have a Paypal account and paid as a guest via debit/credit. I made the purchase before testing out the site because when I added my sources it was over 50 channels and  they only let you generate an EPG for 50 channels with the trial version.

<img width="730" alt="Screenshot 2023-03-15 at 1 53 29 AM" src="https://user-images.githubusercontent.com/37257836/225257000-ed707386-ed4e-40b1-bf44-e9c2050ba32d.png">

**Comment by BellezaEmporium** [comment]
This topic needs to be moved to iptv-org/awesome-iptv.
